plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -530,11 +531,9 @@ func (c *CloudWatch) BuildMetricDatum(point telegraf.Metric) []*cloudwatch.Metri
 }
 
 // Make a list of Dimensions by using a Point's tags. CloudWatch supports up to
-// 10 dimensions per metric so we only keep up to the first 10 alphabetically.
+// 30 dimensions per metric so we only keep up to the first 30 alphabetically.
```

Comment:
Is there a GitHub issue that this is resolving?
It would be nice to link to more details on why this change from 10 to 30 is being done.
Did CloudWatch backend announce they can support 30 dimensions per metric?
If so, any idea why they chose 30 instead of 40, 50, etc.?